spec/requests/users_controller_spec.rb

```diff
@@ -132,6 +143,10 @@
           it "does not allow reactivating the user" do
             expect(page).not_to have_link("Reactivate user", href: "/users/#{user.id}/reactivate")
           end
+
+          it "does not allow resending invitation emails" do
+            expect(page).not_to have_link("Resend invite link", href: "/users/#{other_user.id}/resend-invite")
```

Comment:
actually for all these tests asserting we expect .not_to etc. we don't need to specify the href, since it's already asserting there is no link like that.